src/Rule/NoPhpOpenTagInCodeBlockPhpDirective.php

```diff
@@ -36,6 +36,7 @@ public function check(Lines $lines, int $number): ?string
             && !RstParser::codeBlockDirectiveIsTypeOf($line, RstParser::CODE_BLOCK_PHP_ANNOTATIONS, true)
             && !RstParser::codeBlockDirectiveIsTypeOf($line, RstParser::CODE_BLOCK_PHP_ATTRIBUTES, true)
             && !RstParser::codeBlockDirectiveIsTypeOf($line, RstParser::CODE_BLOCK_PHP_SYMFONY, true)
+            && !RstParser::codeBlockDirectiveIsTypeOf($line, RstParser::CODE_BLOCK_PHP_STANDALONE, true)
```

Comment:
Out of scope, but do you have a plan for this condition in many files ?

Reply:
I tried the DirectiveTrait inPhpCodeBlock method yesterday but it wasn't working 

You can give it a try with this method or introduce a helltet class for this